airbyte-integrations/connectors/destination-databricks/metadata.yaml

```diff
@@ -38,6 +36,7 @@ data:
     sl: 100
     ql: 100
   supportLevel: community
+  supportsRefreshes: true
```

Comment:
Is this the flag that will also cause the platform to send trace messages? Or is that something else?

Reply:
yep, that's all controlled by this flag